src/StoreKeeper/WooCommerce/B2C/FileExport/ProductFileExport.php

```diff
@@ -399,7 +465,21 @@ private function exportStock(array $lineData, WC_Product $product): array
     private function exportImage(array $lineData, WC_Product $product): array
     {
         foreach ($this->getProductImageIds($product) as $index => $imageId) {
-            $lineData["product.product_images.$index.download_url"] = wp_get_attachment_url($imageId);
+            $imageUrl = wp_get_attachment_url($imageId);
+            $lineData["product.product_images.$index.download_url"] = false === $imageUrl ? '' : $imageUrl;
```

Comment:
maybe we can check if [it's valid url](https://www.w3schools.com/php/filter_validate_url.asp) here ? it would fixed the issue with `no` we got in byolga

Reply:
Added checking of valid URL.